google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py

```diff
@@ -45,9 +45,8 @@ def make_entry_for_folder(
 
         entry = datacatalog.Entry()
 
-        # The root folder does not have an ``_id`` field.
-        folder_id = folder_metadata['_id'] if folder_metadata.get(
-            '_id') else folder_metadata.get('name')
+        # The root folder's ``oid`` field is not fulfilled.
+        folder_id = folder_metadata.get('oid') or folder_metadata.get('name')
```

Comment:
What's the different between the `oid` and `_id` fields? Is this relevant info for the users of the connector?

Reply:
I didn't find any information in regards to these fields in the API docs. David (from Karhoo) asked Sisense folks for clarification, but I realized `_id` is pretty much unuseful for the connector and its users, at least for now. After analyzing a couple of API responses, I realized Sisense uses `oid` to build parent-child relationships between folders and dashboards and to build URIs as well. This is why I'm replacing `_id` for `oid`.